swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = 0.0, age = 0):
+        super().__init__( category = "", condition = 0.0, age = 0)
+        self.category = category
+        self.condition = condition
+        self.age = age
```

Comment:
This setup makes it possible to pass in a value for category that is not "Clothing", which could result in an instance of `Clothing` that has some other category.  Right now the values being passed to the `__init__` function of the parent class are default values, so lines 6-8 are necessary to initialize this instance with the values that are being passed in to the constructor.  However, changing the values that are being passed into the super constructor makes it possible to use the code in the parent class, and then lines 6-8 will no longer be necessary.
```suggestion
    def __init__(self, condition = 0.0, age = 0):
        super().__init__( category = "Clothing", condition = condition, age = age)
```
or
```suggestion
    def __init__(self, condition = 0.0, age = 0):
        super().__init__( "Clothing", condition, age)
```